packages/debugger/test/txlog.js

```diff
@@ -112,6 +112,16 @@ function byName(variables) {
   );
 }
 
+function verifyNoIntermediates(untied) {
```

Comment:
Oh, so, to make the tests easier to write, I had this check operate on `txlog.proc.transactionLog` (the untied version, with the nodes indexed by JSON pointer) rather than `txlog.views.transactionLog` (with everything in a tree that would have to be walked).  Since the tree structure isn't really relevant here, I figured it was easier to just operate on the untied version.

Reply:
Ah, thanks!